docs/website/root/manual/getting-started/bootstrap-cardano-node.md

```diff
@@ -236,21 +236,23 @@ List the available snapshots with which you can bootstrap a Cardano node:
-| 72    | 1377      | 7cea14e5742387ca770a74f3e3cfdd93fc38573e2babbf05df292888a528ab15 | 1256695921 |         1 | 2023-05-31T01:56:38.178640636Z |
-+-------+-----------+------------------------------------------------------------------+------------+-----------+--------------------------------+
-| 72    | 1376      | 4cb153b55f6cadf47bc48cc3112c16a037ee49416820190dc9e121d4aa49369f | 1255258102 |         1 | 2023-05-30T20:11:54.620669432Z |
-+-------+-----------+------------------------------------------------------------------+------------+-----------+--------------------------------+
-|       |           |                                                                  |            |           |                                |
++-------+-----------+------------------------------------------------------------------+------------+-----------+--------------------------------+----------------------+
+| Epoch | Immutable | Digest                                                           | Size       | Locations | Created                        | Cardano node version |
++-------+-----------+------------------------------------------------------------------+------------+-----------+--------------------------------+----------------------+
+| 72    | 1379      | a305aa11b0e2ccf737d4f5def8b0a9f2245eded2b4ec4be876f7bd64deddcbbf | 1259745182 |         1 | 2023-05-31T14:02:40.150189810Z | 8.1.2                |
++-------+-----------+------------------------------------------------------------------+------------+-----------+--------------------------------+----------------------+
+| 72    | 1378      | 5c2214b0b3a00cccacc96b65f9741d4e818df0bc092bee30986e4d554396c6fd | 1258142105 |         1 | 2023-05-31T07:48:08.357263836Z | 8.1.2                |
++-------+-----------+------------------------------------------------------------------+------------+-----------+--------------------------------+----------------------+
+| 72    | 1377      | 7cea14e5742387ca770a74f3e3cfdd93fc38573e2babbf05df292888a528ab15 | 1256695921 |         1 | 2023-05-31T01:56:38.178640636Z | 8.1.2                |
++-------+-----------+------------------------------------------------------------------+------------+-----------+--------------------------------+----------------------+
+| 72    | 1376      | 4cb153b55f6cadf47bc48cc3112c16a037ee49416820190dc9e121d4aa49369f | 1255258102 |         1 | 2023-05-30T20:11:54.620669432Z | 8.1.2                |
++-------+-----------+------------------------------------------------------------------+------------+-----------+--------------------------------+----------------------+
+|       |           |                                                                  |            |           |                                |			|
 …
 ```
 
+:warning: If you restore a node with a different version range than the one advertised, it may cause extra time to restore the node due to ledger computations or even crash the node.
```

Comment:
```suggestion
:warning: If you restore a Cardano node with a version not included in the advertised range of compatible versions, it may cause extra time to restore the node due to ledger computations, or even crash the node.
```